### DISCUSSION TITLE
Call to a member function requestedRelations() on null

### DISCUSSION THREAD

**Discussion by aggarwalarpit93** [tailflow/laravel-orion#286] [Help]
Hello, I am using Laravel Orion for my API requests. I am calling store method to store the user in the database and I am getting this error:

Call to a member function requestedRelations() on null

Here is my controller code.

```
protected $model = User::class;

protected function buildIndexFetchQuery(Request $request, array $requestedRelations): Builder
    {
        return parent::buildIndexFetchQuery($request, $requestedRelations);
    }
```

**Comment by alexzarbn** [comment]
Hi @aggarwalarpit93,

Please share the payload of your request.
Also, what Orion and Laravel version are you using?
